### PR TITLE
Add name and update supported platfroms for Go & PHP

### PR DESCRIPTION
**Pull request by cleptric**

### CODE REVIEW

File: src/platforms/common/enriching-events/identify-user.mdx [hunk 1]
```diff
@@ -31,11 +31,18 @@ Your internal identifier for the user.
 
 The username. Typically used as a better label than the internal id.
 
+<PlatformSection supported={["go"]}>
```
Comment (marandaneto) [inline src/platforms/common/enriching-events/identify-user.mdx:34]: Dart and Flutter as well

File: src/platforms/common/enriching-events/identify-user.mdx [hunk 2]
```diff
@@ -31,11 +31,18 @@ Your internal identifier for the user.
 
 The username. Typically used as a better label than the internal id.
 
+<PlatformSection supported={["go", "dart"]}>
+
+### `name`
+
+Human readable name of the user.
+</PlatformSection>
```
Comment (lizokm) [inline src/platforms/common/enriching-events/identify-user.mdx:39]:
```suggestion
<PlatformSection supported={["go", "dart"]}>

### `name`

The user's first and/or last name.
</PlatformSection>
```

**Review by lizokm — APPROVED** [review]
@cleptric "human readable name" has an odd ring to it. Let me know if my edit works.

File: src/platforms/common/enriching-events/identify-user.mdx (hunk 2 above)
Comment (cleptric) [reply src/platforms/common/enriching-events/identify-user.mdx:39]: Yep, this sounds way better, thank you!
I got this from our realy code, https://github.com/getsentry/relay/blob/e4d11ee25bb1b85eeec9ab4fbdee463248eb5b40/relay-general/src/protocol/user.rs#L59 😬

**Pull request merged**